src/treetensornetworks/opsum_to_ttn.jl

```diff
@@ -221,11 +233,354 @@ function svdTTN(
   return H
 end
 
+function qn_svdTTN(
```

Comment:
I would actually call this `ttn_svd`, since it is the SVD backend of the TTN constructor.

Reply:
I'll take care of this as soon as I remove the deprecated parts of the code.